### PR TITLE
updated release notes

### PR DESCRIPTION
**Pull request by anallathambi**
updated release notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1138,6 +1138,12 @@ The `jumpstation` connection options support the same values (for as much as it
 
 <a name="release_history"></a>
 # Release History
+* Overthere 5.0.21 (07-Oct-2020)
+    * Upgrade to SMBJ 0.10.0 and Make artifact-copy-strategy work on satellite
+* Overthere 5.0.20 (18-Sep-2020)
+    * Fix for #247 (Make artifact-copy-strategy work on satellite)
```
Comment (racheruk) [inline README.md:1144]: This can be removed. Mentioning it once for both versions is fine.

File: README.md [hunk 2]
```diff
@@ -1138,6 +1138,12 @@ The `jumpstation` connection options support the same values (for as much as it
 
 <a name="release_history"></a>
 # Release History
+* Overthere 5.0.21 (07-Oct-2020)
+    * Upgrade to SMBJ 0.10.0 and Make artifact-copy-strategy work on satellite
```
Comment (racheruk) [inline README.md:1142]: Version changes are cumulative. That means we dont have to mention " and Make artifact-copy-strategy work on satellite" on this line.

File: README.md [hunk 3]
```diff
@@ -1138,6 +1138,10 @@ The `jumpstation` connection options support the same values (for as much as it
 
 <a name="release_history"></a>
 # Release History
+* Overthere 5.0.21 (07-Oct-2020)
+    * Make artifact-copy-strategy work on satellite
```
Comment (racheruk) [inline README.md:1142]: No mention of smbj upgrade ?

Comment (anallathambi) [reply README.md:1142]: Previous review cmts: Version changes are cumulative. That means we don't have to mention " and Make artifact-copy-strategy work on satellite" on this line. 
Just removed version .

Comment (racheruk) [reply README.md:1142]: There is no mention of SMBJ upgrade ?

**Pull request merged**